src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -319,6 +321,145 @@ def fuzz(self,
                                 artifact_prefix, additional_args)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  def __init__(self, executable_path, default_args=None):
+    self.fuchsia_bucket_name = 'fuchsia_on_clusterfuzz_resources_v1'
+
+    # This initialization routine assumes that the GCS bucket contains the
+    # standard Fuchsia SDK, as well as:
+    # * /qemu-for-fuchsia/*
+    # * /.ssh/*
+    resources_path = os.path.join(os.getcwd(), self.fuchsia_bucket_name)
+
+    shell.create_directory_if_needed(resources_path)
+
+    if 'FUCHSIA_RESOURCES_PATH' not in os.environ:
+      logs.log_error(
+          ('Could not find path for remote Fuchsia resources bucket'
+           '(FUCHSIA_RESOURCES_PATH)!')
+      )
+
+    gsutil_command_arguments = [
+        'cp', '-r',
+        environment.get_value('FUCHSIA_RESOURCES_PATH', ''), resources_path
```

Comment:
Why call `environment.get_value('FUCHSIA_RESOURCES_PATH', '')` instead of `environment.get_value('FUCHSIA_RESOURCES_PATH')` since we know it will always be in environ.
I would do this btw:
```
fuchsia_resources_url = environment.get_value('FUCHSIA_RESOURCES_URL')
if fuchsia_resources_url is None:
  logs.log_error(
          ('Could not find path for remote Fuchsia resources bucket'
           '(FUCHSIA_RESOURCES_URL)!')
      )
      return  # Bail
gsutil_command_arguments = ['cp', '-r', gcs_resources_url, resources_path]
```
(I think "url" is more descriptive than "path")

Reply:
done